src/ui/FacetsMobileMode/FacetsMobileMode.ts

```diff
@@ -39,15 +39,25 @@ export class FacetsMobileMode extends Component {
      */
     isModal: ComponentOptions.buildBooleanOption({ defaultValue: false }),
     /**
-     * Whether to display a background behind the facets when the end user expands them in mobile responsive mode.
+     * Whether to display a transparent overlay behind the facets when the end user expands them in mobile responsive mode.
+     *
+     * By default, the following behavior applies:
+     * - `true` when [isModal]{@link FacetsMobileMode.options.isModal} is `false`
+     * - `false` when [isModal]{@link FacetsMobileMode.options.isModal} is `true`
      */
-    showBackgroundWhileOpen: ComponentOptions.buildBooleanOption({
-      postProcessing: (value, options) => (Utils.isNullOrUndefined(value) ? !options.isModal : value)
+    displayOverlayWhileOpen: ComponentOptions.buildBooleanOption({
+      postProcessing: (value, options: IFacetsMobileModeOptions) => (Utils.isNullOrUndefined(value) ? !options.isModal : value)
     }),
     /**
      * Whether to disable vertical scrolling on the specified or resolved [`scrollContainer`]{@link FacetsMobileMode.options.scrollContainer} while facets are expanded in mobile responsive mode.
+     *
+     * By default, the following behavior applies:
+     * - `true` when [isModal]{@link FacetsMobileMode.options.isModal} is `true`
+     * - `false` when [isModal]{@link FacetsMobileMode.options.isModal} is `false`
      */
-    lockScroll: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    preventScrolling: ComponentOptions.buildBooleanOption({
+      postProcessing: (value, options: IFacetsMobileModeOptions) => (Utils.isNullOrUndefined(value) ? options.isModal : value)
```

Comment:
@btaillon I don't think this will be breaking since isModal isa new option too, and false by default